### DISCUSSION TITLE
Unable to use `NOT repo:name`

### DISCUSSION THREAD

**Discussion by jaredallard** [community/community#11197] [Other Feature Feedback, Questions, & Ideas] [answered]
When attempting to use `NOT repo:name` I get "search failed" every single time :(

**Comment by look** [comment, marked as answer]
Is there a public example you can share with us? Does the search work for you if you use `NOT repo:owner/name`? Thanks.

**Reply by zp-zpanda** [reply]
I've just tested it and using `NOT repo:owner/name` works but `NOT repo:name` doesn't. (Tested with `torvalds/linux`)

**Reply by twpayne** [reply]
I got the "search failed" error with the query:

    k8s.io/kubernetes/test/e2e NOT repo:kubernetes
